### ISSUE TITLE
Unclear how to start using (this) preview feature

### ISSUE THREAD

**Issue by maggiepint** [Azure/azure-cli-extensions#976]
@michaelw85 commented on [Mon Sep 30 2019](https://github.com/Azure/azure-cli/issues/10708)

It took me quite some time to find a blog where it shown to use `az extension add --name azure-devops` to be able to start using the az pipelines command. It would be nice if there would be some explanation that this is part of the azure-devops extension and should be installed.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 66f0311f-8316-ea52-6171-9eb14b9f7bc7
* Version Independent ID: e79f2bc9-543d-9a14-a08c-911d037e108a
* Content: [az pipelines](https://docs.microsoft.com/en-us/cli/azure/pipelines?view=azure-cli-latest#code-try-1)
* Content Source: [latest/docs-ref-autogen/pipelines.yml](https://github.com/MicrosoftDocs/azure-docs-cli/blob/live/latest/docs-ref-autogen/pipelines.yml)
* GitHub Login: @rloutlaw
* Microsoft Alias: **routlaw**

**Comment by atbagga** [comment]
@geverghe Simple way to fix it could be adding a long summary in help of the root level groups - "Part of azure-devops extension." That way it will show up at the top of all these command references at the top of the page.

**Comment by geverghe** [comment]
>It would be nice if there would be some explanation that this is part of the azure-devops extension and should be installed.

@maggiepint - to ensure we meet your expectation, could you tell me which information source you were referring to for this comment (the places you looked at didn't find this information)? 

@atbagga - That makes sense. Today the summary for azure-devops in `az extension list-available`  says " Tools for managing Azure DevOps."; We can change this to "Tools for managing Azure DevOps - includes az pipelines, az repos, az artifacts, az boards and az devops command groups".

Based on @maggiepint 's response, I want to ensure we cover all the entry points to help/documents (which I think it already does) to have the same message.

**Comment by michaelw85** [comment]
@geverghe I placed my comment in the azure cli pages, I don't see my original question anymore but I thought I placed it here: https://docs.microsoft.com/en-us/cli/azure/ext/azure-devops/pipelines?view=azure-cli-latest (or on one of the child pages)

**Comment by geverghe** [comment]
@michaelw85 - Thank you for the clarification - provides more context on the issue. Including a reference of the page
![image](https://user-images.githubusercontent.com/16950956/66491259-0c2ec680-ead0-11e9-9dae-bd381e9b24e2.png)

Proposed Update:
# az pipelines
Manage Azure Pipelines. This command group is a part of the azure-devops extension.

Let me know if that works.

Two quick clarification questions to better understand your scenario:
1. Could you tell me how you landed on the pipelines command page? Google/some other starting point?
2. The intent of the tree hierarchy on the left was to show case the extensions and the command groups beneath it and thereby enable users to see which command belongs to what extension.
Could you tell me if you missed this? Or looked at this, but the tree was not conveying the intended message?

Any feedback to improve our documentation page would be super useful.

**Comment by michaelw85** [comment]
1) I always use google to get to the correct az cli documentation. In this case I was specifically searching if it was possible to create a variable group using the cli.
2) I noticed yesterday when looking up the page that the hierarchy shows the module name. This did not stand out to me before. I think I now made the connection since I used the install command for the first time. Now that I know it's really clear but I didn't even know that additional extensions could/should be installed. Up until this point I was just using some command which are available by default apparently.
